osgi-dependencies/aet-features.xml

```diff
@@ -47,6 +47,7 @@
         <bundle>mvn:commons-collections/commons-collections/3.2.1</bundle>
         <bundle>mvn:commons-fileupload/commons-fileupload/1.3.2</bundle>
         <bundle>mvn:commons-io/commons-io/2.4</bundle>
+        <bundle>mvn:commons-lang/commons-lang/2.6</bundle>
```

Comment:
Is that new lib? Where it is or will be used?

Reply:
It is not new: it is currently available on Karaf, but on the new one it is missing.
Putting it here doesn't change anything for current version.

Reply:
Yes, I think this is a good idea if the dependency is only in AET code.